src/analysis_and_optimization/Pedantic_dist_warnings.ml

```diff
@@ -106,18 +104,16 @@ let value_out_of_range (range : range) (v : float) =
     | None -> false in
   lower_bad || upper_bad
 
-(** Check for inconsistency between a distribution argument's constraint and
-   a literal value *)
+(** Check for inconsistency between a distribution argument's constraint and a
+    literal value *)
 let value_mismatch_constraint (constr : var_constraint) (v : float) =
   match constr with
   | Range range -> value_out_of_range range v
   (* We don't know how to check if a value falls into a constraint other than
      Range, unless we want to inspect e.g. matrix literals. *)
   | _ -> false
 
-(*********************
-   Argument constraint mismatch warnings
- ********************)
+(** Argument constraint mismatch warnings *)
```

Comment:
This comment has lost `***********`. That's fine by me but several other ****-highlighted comments that have kept the markers. Is there some logic to these formatting decisions?